docs/src/_getting-started/part1.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Tutorial Part 1 - Extract Data
```

Comment:
Meta question - since this isn't in the tutorial section, do we want it to be a walkthrough? How are you thinking about that structure?

Reply:
It's still a tutorial. The way I think about it is:
- "Getting Started" is a section which hosts just one tutorial. "The Meltano Tutorial".
- Then we add an "Advanced Tutorials" section.

Fwiw, I don't think the naming is that important, but rather the intent. I'd aim for having both Getting Started & Advanced Tutorials as tutorial sections (in the diataxis framework).

Reply:
@sbalnojan makes sense! 

Also, don't forget to tag me in basically every comment, otherwise I won't see it. I only saw this because you re-requested a review, so it worked out 😄

Reply:
@tayloramurphy will do.